ush/parsing_namelists_WW3.sh

```diff
@@ -114,213 +104,60 @@ WW3_namelists(){
-                                     ww3_multi.inp.tmpl | \
-  sed -n "/DUMMY/!p"               > ww3_multi.inp
-
-  rm -f ww3_multi.inp.tmpl buoy.loc
-
-  cat ww3_multi.inp
-
-else 
-  #ww3_shel 
-
 # Initialize inp file parameters
-  ICELINE='F F'
-  CURRLINE='F F'
-  WINDLINE='F F'
-
-  case ${WW3ATMINP} in
-    'YES' )
-      WINDLINE="T F";;
-    'CPL' )
-      WINDLINE="C F";;
-  esac
-
-  case ${WW3ICEINP} in
-    'YES' )
-      ICELINE="T F";;
-    'CPL' )
-      ICELINE="C F";;
-  esac
-
-  case ${WW3CURINP} in
-    'YES' )
-      CURRLINE="T F";;
-    'CPL' )
-      CURRLINE="C F";;
-  esac
-
-  sed -e "s/IOSRV/${IOSRV}/g" \
-      -e "s/OUTPARS/${OUTPARS_WAV}/g" \
-      -e "s/ICELINE/$ICELINE/g" \
-      -e "s/CURRLINE/$CURRLINE/g" \
-      -e "s/WINDLINE/$WINDLINE/g" \
-      -e "s/RUN_BEG/$time_beg/g" \
-      -e "s/RUN_END/$time_end/g" \
-      -e "s/OUT_BEG/$time_beg_out/g" \
-      -e "s/OUT_END/$time_end/g" \
-      -e "s/DTFLD/ $DTFLD_WAV/g" \
-      -e "s/GOFILETYPE/ $GOFILETYPE/g" \
-      -e "s/POFILETYPE/ $POFILETYPE/g" \
-      -e "s/DTPNT/ $DTPNT_WAV/g" \
-      -e "s/DTPNT/ $DTPNT_WAV/g" \
-      -e "/BUOY_FILE/r buoy.loc" \
-      -e "s/BUOY_FILE/DUMMY/g" \
-      -e "s/RST_BEG/$time_rst_ini/g" \
-      -e "s/RSTTYPE/$RSTTYPE_WAV/g" \
-      -e "s/RST_2_BEG/$time_rst2_ini/g" \
-      -e "s/DTRST/$DT_1_RST_WAV/g" \
-      -e "s/DT_2_RST/$DT_2_RST_WAV/g" \
-      -e "s/RST_END/$time_rst1_end/g" \
-      -e "s/RST_2_END/$time_rst2_end/g" \
-                                     ww3_shel.inp.tmpl | \
-  sed -n "/DUMMY/!p"               > ww3_shel.inp
-
-  rm -f ww3_shel.inp.tmpl buoy.loc
-
-  cat ww3_shel.inp
-
-fi   
+ICELINE='F F'
+CURRLINE='F F'
+WINDLINE='F F'
+
+case ${WW3ATMINP} in
+  'YES' )
+    WINDLINE="T F";;
+  'CPL' )
+    WINDLINE="C F";;
+esac
+
+case ${WW3ICEINP} in
+  'YES' )
+    ICELINE="T F";;
+  'CPL' )
+    ICELINE="C F";;
+esac
+
+case ${WW3CURINP} in
+  'YES' )
+    CURRLINE="T F";;
+  'CPL' )
+    CURRLINE="C F";;
+esac
+
+sed -e "s/IOSRV/${IOSRV}/g" \
+    -e "s/OUTPARS/${OUTPARS_WAV}/g" \
+    -e "s/ICELINE/$ICELINE/g" \
+    -e "s/CURRLINE/$CURRLINE/g" \
+    -e "s/WINDLINE/$WINDLINE/g" \
+    -e "s/RUN_BEG/$time_beg/g" \
+    -e "s/RUN_END/$time_end/g" \
+    -e "s/OUT_BEG/$time_beg_out/g" \
+    -e "s/OUT_END/$time_end/g" \
+    -e "s/DTFLD/ $DTFLD_WAV/g" \
+    -e "s/GOFILETYPE/ $GOFILETYPE/g" \
+    -e "s/POFILETYPE/ $POFILETYPE/g" \
+    -e "s/DTPNT/ $DTPNT_WAV/g" \
+    -e "s/DTPNT/ $DTPNT_WAV/g" \
+    -e "/BUOY_FILE/r buoy.loc" \
+    -e "s/BUOY_FILE/DUMMY/g" \
+    -e "s/RST_BEG/$time_rst_ini/g" \
+    -e "s/RSTTYPE/$RSTTYPE_WAV/g" \
+    -e "s/RST_2_BEG/$time_rst2_ini/g" \
+    -e "s/DTRST/$DT_1_RST_WAV/g" \
+    -e "s/DT_2_RST/$DT_2_RST_WAV/g" \
+    -e "s/RST_END/$time_rst1_end/g" \
+    -e "s/RST_2_END/$time_rst2_end/g" \
+                                   ww3_shel.inp.tmpl | \
+sed -n "/DUMMY/!p"               > ww3_shel.inp
+
+rm -f ww3_shel.inp.tmpl buoy.loc
+
+cat ww3_shel.inp
```

Comment:
This seems simple and logical on the surface but is conditioned based on the assumption that the `else` part of this `if-else` block is correct and the `ww3_shel.inp.tmpl` has been kept up-to-date.  
Might need some guidance here as well.